packages/framework-core/src/booster-graphql-dispatcher.ts

```diff
@@ -28,36 +29,38 @@ export class BoosterGraphQLDispatcher {
 
     this.graphQLSchema = new GraphQLGenerator(config, commandDispatcher, this.readModelDispatcher).generateSchema()
     this.websocketHandler = new GraphQLWebsocketHandler(
+      config,
       logger,
-      this.config.provider.readModels.notifySubscription.bind(null, config),
+      this.config.provider.auth,
+      this.config.provider.connections,
```

Comment:
Now we are passing the `auth` related functions to the GraphQLWebSocketHandler so that it is able to retrieve a user from a token. 

Also, we are passing the new `connections` related function to be able to store, fetch and delete connection data (user information)